R/acf.R

```diff
@@ -63,7 +63,10 @@ ACF <- function(.data, ..., lag_max = NULL, demean = TRUE,
   compute_acf <- function(.data, value, ...){
     value <- enexpr(value)
     x <- eval_tidy(value, data = .data)
-    acf <- tail(as.numeric(acf(x, plot=FALSE, ...)$acf), -1)
+    acf <- as.numeric(acf(x, plot=FALSE, ...)$acf)
+    if(type[1] != "partial"){ # First indx already dropped if partial
```

Comment:
Works well now. However overnight I had a remembered that `match.arg()` allows for partial matching.
I think the safest thing to do would be to call `match.arg()` in `ACF()` and then the type comparison is simpler (`type ~= "partial")

Reply:
Right you are. Now it's starting to become embarrassing. ;) Pushed once again.